### PR TITLE
Fix eclipse warnings

### PR DESCRIPTION
**Pull request by dpursehouse**

### CODE REVIEW

File: src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java [hunk 1]
```diff
@@ -172,8 +172,6 @@
     private String serverName;
     private String gerritSlaveId;
     private List<PluginGerritEvent> triggerOnEvents;
-    @Deprecated
-    private transient boolean allowTriggeringUnreviewedPatches;
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java]: :-1: This will break loading of old configurations.

Comment (dpursehouse) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java]: Fixed in the follow up commit cb3c5b565773c9c0ab8032b86c9e68a801d1211f.

File: src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java [hunk 2]
```diff
@@ -302,7 +302,7 @@ public GerritTrigger(
         this.triggerConfigURL = triggerConfigURL;
         this.gerritTriggerTimerTask = null;
         triggerInformationAction = new GerritTriggerInformationAction();
-        this.allowTriggeringUnreviewedPatches = allowTriggeringUnreviewedPatches;
+        this.allowTriggeringUnreviewedPatches = false;
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java]: just remove this line completely and put ```@SuppressWarnings("unused")``` on the declaration instead.

Comment (dpursehouse) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java]: Done with 32eb41144fd757743a3656325e56f47a3d402187

**Pull request merged**